posts/enterprise-lacking-cloud-skills.md

```diff
@@ -0,0 +1,63 @@
+
+Cloud providers are working hard to provide far greater security than that of on-premise. Public cloud is making great progress with [Capital One](https://aws.amazon.com/solutions/case-studies/capital-one/) and [Pacific Life](https://aws.amazon.com/solutions/case-studies/pacific-life-insurance/) being a big case studies for AWS. Azure and Google are also building their own case studies in this space that has up till now been wary of public cloud.
+
+Researchers for Deutsche Bank are [calling 2017 the year of cloud migration](https://blogs.wsj.com/cio/2016/06/09/big-banks-starting-to-embrace-public-cloud-deutsche-bank-says/) for financial institutions. If the use cases are supported by the technology now, it's surely more a battle of politics and resourcing for highly regulated industries.
+
+## Cloud performance is achieved with new techniques
+
+An issue we see time and time again is the fitting of a square peg into a round hole. Whilst certainly tempting to try and run your applications within a cloud environment this will certainly lead to lots of head scratching.
+
+There are new fundamentals for building applications for the cloud. That's not to say everything needs to be rewritten but they must be adapted to operate in the cloud.
+
+Workloads need to be handled differently too. No more are the days of "scale up" which is the technique of adding more RAM or CPU to your servers to handle larger amounts of work. "Scale out" is now the preferred mode of achieving performance at a lower cost. This often means breaking down larger workload applications into smaller ones which represent individual tasks rather than a whole data pipeline.
+
+Cloud offers a different model for seeking performance. Issues are encountered when trying to apply the monolithic application approach of the past to an infrastructure that is better utilised when broken down into simpler pieces. Again, at the core of this issue is the lack of knowledge on how to approach the opportunity provided by cloud.
+
+## Migrations shouldn't last years but months at most
+
+When organisations rightly recognise their lack of skills, purchasing a huge department of engineers (often 50+ engineers) from service providers becomes tempting. A huge amount of legacy requires a huge amount of engineering power right?
+
+It turns out the decision to purchase multi year migrations from oil tanker sized service providers suffers from a few problems too. Firstly, the old guard of service providers who still think throwing 50 engineers at a problem will be faster than a team of 5 are equally ill-equipped in skills to migrate to the cloud.
+
+Just as workloads needing to be broken down for migration to the cloud so do the programmes of work. Instead of migrating everything at once or even just a collection of applications, build a roadmap for migrating one. Commit to the spend of migrating only that one piece of the puzzle.
```

Comment:
This sentence is a bit confusing: "Instead of migrating everything at once or even just a collection of applications, build a roadmap for migrating one." Is it weighing up migrating multiple migrations versus single applications? Not sure if that's right, but if it is you could use: 
"Instead of migrating multiple applications at once, build a roadmap for migrating a single application."